tests/helpers.py

```diff
@@ -183,7 +197,7 @@ def __init__(
         # OVS and controlled by a remote controller
         patch('mininet.util.fixLimits', side_effect=None)
         self.net = Mininet(
-            topo=topos.get(topo_name, (lambda: RingTopo()))(),
+            topo=topos.get(topo_name, topos[topo_name])(),
```

Comment:
This should use the `RingTopo` if an invalid `topo_name` is used. With this change, a `KeyError` will be raised for an invalid `topo_name`. You should keep the original code or change it to `topo=topos.get(topo_name, topos["ring"])`

Reply:
Ok.

Reply:
yes, agreed!